server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/redirect/NotifyMailetsMessageTest.java

```diff
@@ -195,6 +195,7 @@ public void generateMessageShouldSpecifySizeInAReadableWay() throws Exception {
         assertThat(generateMessage).contains("Size: 5.9 KiB");
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
why ?

Reply:
@aduprat  ?

Reply:
It's on the _when_ line:
```
when(mail.getMessageSize()).thenThrow(MessagingException.class);
```
the warning is:
```
Type safety: A generic array of Class<? extends Throwable> is created for a varargs parameter
```

Reply:
Ok let's keep it. Would that be Ok matthieu?

Reply:
no that's not ok, please use : 
```
        when(mail.getMessageSize()).thenThrow(new MessagingException());
```